Source/ISHRemote/Trisoft.ISHRemote/Connection/InfoShareWcfSoapWithOpenIdConnectConnection.cs

```diff
@@ -17,31 +17,20 @@
 using System;
 using System.Collections.Generic;
 using System.IdentityModel.Tokens;
-using System.Linq;
 using System.Net.Http;
 using System.ServiceModel;
-using System.ServiceModel.Description;
 using System.Xml.Linq;
-using System.Xml.XPath;
 using Trisoft.ISHRemote.Interfaces;
 using System.ServiceModel.Security;
 using System.Security.Cryptography.X509Certificates;
 using System.Net;
-using System.IdentityModel.Selectors;
 using System.Net.Http.Headers;
-using IdentityModel.Client;
-using IdentityModel.OidcClient;
-using System.Threading.Tasks;
-using System.Threading;
 using System.Security.Claims;
 using System.Text;
 using System.Xml;
 using System.IO;
 
 #if NET48

```

Comment:
Change empty pragma if with still something in the else into `#if !NET48` and remove the `#else`